services/quaranteam/Dnd/Connections.java

```diff
@@ -0,0 +1,55 @@
+package strategyPattern;
+
+import java.util.Scanner;
+
+public class Connections {
+
+	public static void main(String[] args) 
+	{
+		
+		Scanner in = new Scanner(System.in);
+		String input = "";
+		String[] splitted;
+		String app;
+		Context c;
+		
+		for (int i = 0; i < 3; i++)
+		{
+			
+		System.out.println("Enter friend's name first and app to connect with second: ");
+		input = in.nextLine();
+		
+		splitted = input.split(" ");
+		
+		app = splitted[1];
+		
+		c= new Context(splitted);
+		
+		
+		switch (app.toLowerCase())
+		{
+		
+		case "facebook":
+				c.changeStrat(new Facebook());
+				c.filter();
+				break;
```

Comment:
odd white spacing

Reply:
This is for a different project, it isn't supposed to be in here, oops. I'll take care of these files